config/crd/bases/redis.redis.opstreelabs.in_redisclusters.yaml

```diff
@@ -2322,19 +2336,6 @@ spec:
             - kubernetesConfig
             type: object
         type: object
-    additionalPrinterColumns:
-    - jsonPath: .spec.clusterSize
-      description: Current master node count
-      name: Master
-      type: integer
-    - jsonPath: .spec.clusterSize
-      description: Current slave node count
-      name: Slave
-      type: integer
-    - jsonPath: .metadata.creationTimestamp
-      description: Last Deployment Time
-      name: Last Deployment Time
-      type: date
```

Comment:
Not sure why this is being removed, nor where its being referenced/defined.

Reply:
This is something the bug with operator-sdk, it removes additional printer coloums. You may have to add this manually